src/Servers/IIS/build/Build.Common.Settings

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Suggest adding comments at the top of these files indicating which build/*.Setting or *.props file should be imported when